### ISSUE TITLE
Final credits scene doesn't show black bars

### ISSUE THREAD

**Issue by mrpenguinb** [HenryJk/PvZWidescreen#6]
![image](https://user-images.githubusercontent.com/13060086/160332404-6744774d-bf77-4fac-81b5-5138510ee45c.png)

**Comment by HenryJk** [comment]
Yes, I'm aware of this issue and haven't yet found a solution for it.
